alert/alert.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root{
+    --primary-color: #34D399;
+    --secondary-color: ;
+    --primary-voilet: #8F00FF;
+    --grey: #EBECED;
+    --primary-dark: #272C33;
+    --secondary-dark: #3F4652;
+    --light-voilet: #cf9cf7;
+    --light-green: #b0f3d4;
+    --light-yellow: #fde68a;
+    --light-red: #fd8181;
+    --light-blue: #b0ddf5;
+    --blue: #0381F7;
+    --red: #DC2626;
+    --green: #059669;
+    --yellow: #f59e0b;
+}
```

Comment:
You should have all of these colors in utility.css, not inside alert.css as you would need these colors at various pages.